### ISSUE TITLE
How can i disable the "some heaters are still on" Message?

### ISSUE THREAD

**Issue by Primus007** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2703]
My goal is to disable the message "some heaters are still on". This message popup on my display when i have preheated the nozzle/bed and go to the Menue Movement -> Bed Level and then Back.

I have a new TFT35 E3 V3 Display. 
The Source is the Vx.x.27 from here. 
I have disabled the SAFETY_ALERT in the configuration.h and compiled and updated the Firmware on the display.
but the  message still popup. 

How can i disable this message? Why it does not work if i disable SAFETY_ALERT?

**Comment by radek8** [comment]
The setting in configuration.h is the default, but parameters in config.ini override it.
Did you use Config.ini?

**Comment by Primus007** [comment]
after the reset of the display i have not used a config.ini
And the SAFETY_ALERT is not in the config.ini. This can i edit only in the configuration.h
the display always shows "BIGTREE_GD_TFT35_E3_V3_0 Firmware V27.x Mar 10 2022 in 120MHz" in the info. shouldn't be the current date if the update was successful?

**Comment by radek8** [comment]
Use the current branch. Master
Everything else is obsolete


![image](https://user-images.githubusercontent.com/46979052/217377074-9891764b-fcd0-4a11-b6bc-411f29af9c93.png)

**Comment by radek8** [comment]
Coment A SAFETY_ALERT 

[BIGTREE_GD_TFT35_V3.0_E3.27.x.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/10679795/BIGTREE_GD_TFT35_V3.0_E3.27.x.zip)

**Comment by radek8** [comment]
When updating, you can see the progress of the update on the display.
After the update, the file extension will change from bin to cur
Also update the ini file, language file, and icon and font folder.

**Comment by Primus007** [comment]
i have tried your bin. it does not update. My display does not update any .bin

Now i have noticed that my display does not update the Firmware at all.
i copy the firmware to an sd card.
then i put the sd card in the display and boot with it.
I don't see an update screen
But the firmware was renamed to .cur
I have tried different SD cards
i have tried precompiled firmware and self compiled. 
all other things will be updated. config.ini, themes and language files. Only the .bin not.

i use this bin: BIGTREE_GD_TFT35_V3.0_E3.27.x.bin

here is a photo from the info screen:
![display](https://user-images.githubusercontent.com/6861874/217500804-54c1f45b-f9c7-44bc-9277-6f8eebc6852b.jpg)

**Comment by radek8** [comment]
Is your display really the GD version?
Can you check the CPU model on the display?
Send a photo of the circuit board.

**Comment by radek8** [comment]
SD card should be 8G or smaller.
Formatted as FAT32 (small capacity cards as FAT)

**Comment by Primus007** [comment]
The display is new. it is the original firmware you see on the picture above.
I check the CUP later and upload a photo.
I have tried different sd cards. 
it is strange that the file is renamed but the firmware is not updated.

**Comment by Primus007** [comment]
Yes it is a GD Display. I have tried different sd cards. it is strange that the file is renamed but the firmware is not updated.
![20230208_121906](https://user-images.githubusercontent.com/6861874/217516141-c0c825c3-3f0a-48ff-9847-d02dcab37acd.jpg)


![2023-2-8 12-18-48](https://user-images.githubusercontent.com/6861874/217516273-2cd689e0-1d0e-4744-9ecf-acd2081f11d4.jpg)

**Comment by radek8** [comment]
Yes, you have the correct version (GD)
So far no one else has complained about the same problem...

**Comment by radek8** [comment]
Yes, you have the correct version (GD) I would try to ask for a statement from the manufacturer. On his website, he occasionally responds to user inquiries.
The manufacturer does not respond to questions on Github.

https://support.biqu3d.com/hc/en-us/sections/4415370318745-TFT35-E3-V3-0

**Comment by Primus007** [comment]
Thanks for your help!

**Comment by radek8** [comment]
https://community.biqu3d.com/forum/40-product-discussion/

**Comment by oldman4U** [comment]
Please help the community and close the ticket once you do not need it anymore. Thank you
